README.md

```diff
@@ -1,2 +1,11 @@
 # comp110-worksheet-C
 Base repository for COMP110 worksheet C
+A) The task the algorithim is preforming is seeing if any elements are duplicated
+B) the worst case scenario would be checking all the elements in list I and J and only finding one duplicate at the end
+C) Because it starts at the same place
+D) The algorithim is twice as fast because it only has to check the elements once
```

Comment:
Important to note here that the original algorithm checks each pair twice (i.e. A==B and B==A)